Argus-Labs/starter-game-template
# feat(starter-game-template): update to use a new version of cardinal …

…that returns tx_hash

Update the starter-game-template to use an updated version of cardinal that returns a tx_hash when creating new transaction and when returning receipts. This PR touches a lot of files, but most of them are due to the package name change for cardinal.

To test/verify: 
- run nakama and cardinal with `mage start`. 
- wait for nakama and cardinal to start
- Run the jsclient from the hidden .jsclient folder with `npm install && node main.mjs`
- open a browser to localhost:7351 to access the Nakama UI.
- From the API explorer, hit the nakama/create-persona endpoint with a user ID of `00000000-0000-0000-0000-000000000000` and a body of `{"persona_tag": "xyzzy"}`
- Observe the response has a tx_hash field.
- Observe the jsclient eventually spits out a receipt for the create persona request, which includes the above tx_hash.
